### ISSUE TITLE
Changing resolution in-game glitches the map-screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#5]
```
What steps will reproduce the problem?
1. Pick a resolution, any resolution.
2. Start a game.
3. Pull up the Options screen and change the resolution.

What is the expected output? What do you see instead?

You'd expect the buttons and status bar to go to where they're supposed to
be for each resolution.  Instead, they stay where they were for the old
one, which means they could be offscreen (large to small) or over the wrong
part of the game map (small to large).

Please use labels and text to provide additional information.
```

Original issue reported on code.google.com by `phil.bor...@gmail.com` on 27 Apr 2008 at 8:08

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Changing owner to funnyman3595, as this is definitely in the new map_loop() 
bits.
```

Original comment by `phil.bor...@gmail.com` on 27 Apr 2008 at 10:50

**Comment by GoogleCodeExporter** [comment]
```
Actually, the line that does that is almost exactly where it used to be.  It's 
just
that it's only one or two lines now, because it pulls the locations from a .dat 
file.
 I'll see if I can spot what's going wrong.
```

Original comment by `funnyman3595` on 28 Apr 2008 at 7:13
* Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Okay, 6 lines, but still not the multiple-per-location mess it used to be.

Fixing the location buttons was easy, since they all calculate their position 
the
same way.  The other buttons and button-users were trickier.  For now, I just 
hacked
in a fix in do_refresh, but the way we specify size and position for buttons 
needs to
be standardized at some point.

Fixed in git (87630bd).  I'm still waiting on issue 6 to commit this and issue 
7 back
to SVN, since I think that will be fairly quick.
```

Original comment by `funnyman3595` on 28 Apr 2008 at 8:55
* Changed state: **Fixed**
